packages/schemas/src/player/gamemodes/general/quests/modes/skywars.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { APIData, removeFormatting } from "@statsify/util";
+import { Field } from "../../../../../metadata";
+import { FormattedGame } from "../../../../../game";
+import { GameQuests } from "../game-quests";
+import { QuestTime, getAmountDuring, questFieldData } from "../util";
+import { add } from "@statsify/math";
+
+export class SkywarsQuests implements GameQuests {
+  @Field(questFieldData)
+  public soloWin: number;
+
+  @Field(questFieldData)
+  public soloKills: number;
+
+  @Field(questFieldData)
+  public doublesWin: number;
+
+  @Field(questFieldData)
+  public doublesKills: number;
+
+  @Field(questFieldData)
+  public labWin: number;
+
+  @Field(questFieldData)
+  public corruptedWin: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "Tokens!" } })
```

Comment:
The leaderboard metadata won't get spread properly here because you overwrite the leaderboard field.